src/components/dialogs/parameters/sensitivity-analysis-parameters.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useCallback } from 'react';
+import { Grid } from '@mui/material';
+import { CloseButton, DropDown, LabelledButton, useStyles } from './parameters';
+import { LineSeparator } from '../dialogUtils';
+
+export const SensitivityAnalysisParameters = ({
+    hideParameters,
+    parametersBackend,
+}) => {
+    const classes = useStyles();
+
+    const [providers, provider, updateProvider, resetProvider] =
+        parametersBackend;
+
+    const updateProviderCallback = useCallback(
+        (evt) => {
+            updateProvider(evt.target.value);
+        },
+        [updateProvider]
+    );
+
+    return (
+        <Grid container className={classes.grid}>
+            <Grid container key="sensiAnalysisProvider">
+                <DropDown
+                    value={provider}
+                    label="Provider"
+                    values={providers}
+                    callback={updateProviderCallback}
+                />
+
+                <Grid container paddingTop={1}>
+                    <LineSeparator />
+                </Grid>
+
+                <Grid
+                    container
+                    className={
```

Comment:
same we can use  useMemo  to store the value of the className, this way you can avoid unnecessary re-rendering.
example: 
```
const className = useMemo(
  () => `${classes.controlItem} ${classes.marginTopButton}`,
  [classes.controlItem, classes.marginTopButton]
)
```